src/huggingface_hub/hf_api.py

```diff
@@ -650,8 +658,14 @@ def list_models(
         r.raise_for_status()
         d = r.json()
         res = [ModelInfo(**x) for x in d]
-        if emissions_threshold is not None:
-            return _filter_emissions(res, *emissions_threshold)
+        if emissions_thresholds is not None:
+            if cardData is None:
+                logging.error(
+                    "Warning! `emissions_threshold` passed without setting `cardData=True`. No filtering can be performed"
+                )
+                return res
```

Comment:
Cool error! I wonder if it wouldn't be even better to raise in that case?

Reply:
Agreed, makes sense. Will toss it in and then merge :smile: